### PR TITLE
Hyrule Town and Royal Crypt from The Minish Cap

### PR DESCRIPTION
**Pull request by Delcatty16**

### CODE REVIEW

File: data/Music/The Legend of Zelda Series/The Minish Cap/Minish Cap - Royal Crypt.meta [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Minish Cap - Royal Crypt
```
Comment (PurpleHato) [inline data/Music/The Legend of Zelda Series/The Minish Cap/Minish Cap - Royal Crypt.meta:1]: The Legend of Zelda: The Minish Cap - Royal Crypt

File: data/Music/The Legend of Zelda Series/The Minish Cap/Minish Cap - Royal Crypt.meta [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Minish Cap - Royal Crypt
+0x3
```
Comment (PurpleHato) [inline data/Music/The Legend of Zelda Series/The Minish Cap/Minish Cap - Royal Crypt.meta:2]: 0x03

**Review by PurpleHato — CHANGES_REQUESTED** [review]
Please add more context in the title in the meta file
Check how these are made for consistency with the PC port where we sort all the songs alphabetically
Also and still for consistancy for your banks add 0x0 before your value

**Comment by DaruniasJoy** [comment]
u got samples on discord thats why i will add it now. but for future reference please make sure to give me the lengths of each track and their loop point next time.

please add these tracks also to the spreadsheet.

**Pull request merged**